ciphers/playfair_cipher.py

```diff
@@ -60,11 +81,26 @@ def generate_table(key: str) -> list[str]:
 
 
 def encode(plaintext: str, key: str) -> str:
+    """encode function encodes the given plaintext by taking
+    two strings, the plaintext and the key, as input
+    and returns an encoded string
+
+    >>> print(encode("Hello", "MONARCHY"))
+    CFSUPM
+    >>> print(encode("attack on the left flank", "EMERGENCY"))
+    DQZSBYFSDZFMFNLOHFDRSG
+    >>> print(encode("Sorry!", "SPECIAL"))
+    AVXETX
+    >>> print(encode("Number 1", "NUMBER"))
+    UMBENF
+    >>> print(encode("Photosynthesis!", "THE SUN"))
+    OEMHQHVCHESUKE
+    """
```

Comment:
```suggestion
    """
    Encode the given plaintext using the Playfair cipher.
    Takes the plaintext and the key as input and returns the encoded string.

    >>> encode("Hello", "MONARCHY")
    'CFSUPM'
    >>> encode("attack on the left flank", "EMERGENCY")
    'DQZSBYFSDZFMFNLOHFDRSG'
    >>> encode("Sorry!", "SPECIAL")
    'AVXETX'
    >>> encode("Number 1", "NUMBER")
    'UMBENF'
    >>> encode("Photosynthesis!", "THE SUN")
    'OEMHQHVCHESUKE'
    """
```
Minor reformatting, also no need to use `print` in doctests